api/proto/teleport/okta/v1/okta_service.proto

```diff
@@ -120,10 +120,13 @@ message CreateIntegrationRequest {
   string scim_token = 3;
   // Enable synchronization of access lists.
   bool enable_access_list_sync = 4;
-  // Enable user synchronization.
+  // Enable user synchronization from Okta to Teleport.
   bool enable_user_sync = 5;
-  // Enable synchronization of app groups.
+  // Enable synchronization of app groups from Okta to Teleport.
   bool enable_app_group_sync = 6;
+  // Enable sync from Teleport to Okta. If disabled it will prevent creating Okta assignments which
+  // in turn will disable JIT Access Requests.
+  bool enable_bidirectional_sync = 10;
```

Comment:
Can we move the declaration at the bootom ? 

Could we move the declaration to the bottom? While grouping fields by name makes sense in some cases, the current proto fields ordering looks like this:```
bool enable_bidirectional_sync = 10;
AccessListSettings access_list_settings = 7;
```
In this context, I don’t think field names should take precedence over the natural proto fields ordering. it’s And I think that we should pioritize numeric ordering over alphabetical grouping for clarity and maintainability